File: Electronics_Shop_17.Services/Database/AccessoryCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Services.Database
{
    public class AccessoryCategory : IHelperId, ISoftDelete
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool isDeleted { get; set; }

    }
}

File: angular17_web/src/app/models/accessory-category/accessory.category.ts
export class AccessoryCategory {
  id!: number;
  name!: string;
  isDeleted!: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.id = data.id;
    this.name = data.name;
    this.isDeleted = data.isDeleted;
  }
}
